smart-contracts/tests/Challenge_6.t.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >=0.6.0 <0.9.0;
+
+import 'forge-std/Test.sol';
+
+import '@smart-contracts/Challenge_6.sol';
+
+contract Challenge6 is Test {
+  using Strings for uint256;
+  Contract c;
+  string baseURI = 'ipfs://QmeSjSinHpPnmXmspMjwiXyN6zS4E9zccariGR3jxcaWtq/';
+
+  function setUp() public {
+    c = new Contract('NFT Example', 'NFTEX', 100);
+    c.setBaseURI(baseURI);
+  }
+
+  /// @notice Checks that the tokenURI is properly constructed
+  /// @param _id The token id
+  function testTokenUri(uint256 _id) public {
```

Comment:
Mooie fuzz test maar deze spoiled wel hoe de concatenation moet niet?

Reply:
De concatenation was deel van challenges 3-4, dus ik neem aan dat ze het al weten tegen de tijd dat ze bij challenge 6 aankomen (Deze test zit niet in de eerdere challenges waar concatenation relevant is)

Dus de enige manier waarop ze hier per ongeluk als een spoiler tegenaan komen is als ze de challenges in een andere volgorde doen for some reason, of ze "cheaten" door naar de solutions folder te kijken bijvoorbeeld maar dat is wat mij betreft ook fair game als dit hun preferred method of learning is.